app/src/main/kotlin/com/intive/picover/main/theme/StatusBar.kt

```diff
@@ -0,0 +1,14 @@
+package com.intive.picover.main.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
+import androidx.compose.ui.graphics.Color
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+@Composable
+fun SetStatusBarColor(color: Color) {
```

Comment:
as the naming convention for compose doesn't recommend using verbs in composable functions, maybe `ColoredStatusBar`?

Reply:
Done